filebeat/input/filestream/internal/input-logfile/manager.go

```diff
@@ -165,7 +165,7 @@ func (cim *InputManager) Create(config *conf.C) (v2.Input, error) {
 
 	settings := struct {
 		ID             string        `config:"id"`
-		CleanTimeout   time.Duration `config:"clean_timeout"`
+		CleanTimeout   time.Duration `config:"clean_inactive"`
```

Comment:
Let's also rename `CleanTimeout` -> `CleanInactive`, so we have less confusion mapping this field to the config parameter.